kukkuu/settings.py

```diff
@@ -78,6 +82,8 @@
     KUKKUU_REMINDER_DAYS_IN_ADVANCE=(int, 1),
     KUKKUU_FEEDBACK_NOTIFICATION_DELAY=(int, 15),
     KUKKUU_NOTIFICATIONS_SHEET_ID=(str, ""),
+    LOGIN_REDIRECT_URL=(str, "/admin/"),
+    LOGOUT_REDIRECT_URL=(str, "/admin/"),
```

Comment:
I'm a bit worried that developers might get confused with OIDC setup vs Keycloak auth setup. If I would be doing something OIDC related, I could easily be fooled that this is some what related. Should we use some kind of naming prefix? For example HELUSERS_ADMIN_LOGIN_REDIRECT_URL or HELUSERS_SOCIAL_AUTH_LOGIN_REDIRECT_URL?

Reply:
I'll merge this now as it is. Let's talk today at some point live about this and see if there's a need for some refactoring related to the environment variable names.